### ISSUE TITLE
Multiples animations at the same time

### ISSUE THREAD

**Issue by joelmora** [adobe-webplatform/Snap.svg#470]
Hi!

I am working on a couple animations, but I wonder if it is possible to have multiple animations running at the same time but at different speeds. 

I create this [jsfiddle](https://jsfiddle.net/bv7e5kjL/) to probe what is happening, I want the balloon to inflate/deflate at 10 second rate, but at the same time add a vibration effect to it, but at different speed. You can uncomment the line `// vibration();` to see that deflate effect doesn't work after apply the vibration effect.

I think it has to do with [this issue](https://github.com/adobe-webplatform/Snap.svg/issues/125) #125   but I am not sure.

I would be glad if you guys give a hand.

**Comment by ibrierley** [comment]
Yes it is possible, however, what you are doing is kind of different. 
You could have 2 animations that animate different attributes...but I think in your case, you are running 2 animations that conflict with each other.

It's a bit like saying I want to animate the balloon blue and red at the same time. It doesn't make sense (because you are changing the transform attribute, not a separate rotation). If you say you want to animate the balloon up and blue, it does make sense.

So this is where it gets a bit confusing with 2 transforms and interpolating at the same time. It will get messy I think :).

What I personally would look at, would be placing 'upper' inside a group for example, eg upperGroup. Then animating upperGroup with the vibration transform, and animating upper with the other transform. That way, they shouldn't conflict.

**Comment by joelmora** [comment]
It worked! Thanks man @ibrierley, it was super simple, but as I am starting with this library I don't know every solution.

Here is the updated [jsfiddle](https://jsfiddle.net/bv7e5kjL/1/)

**Comment by clementgonzalez** [comment]
The "group" solution is more an _hack_ than a real solution.
There's is nothing weird trying to have a shape that stretch horizontally in 2s, and vertically in 1s.
I Thought I would be possible to adapt for animations [this solution](http://svg.dabbles.info/snaptut-pluginrelativetransform), but I didn't succeed.  May be there's way ????

**Comment by ibrierley** [comment]
Not quite sure why you are trying though, this was a long time ago, unless you have a separate problem you are trying to resolve (in which case I'd post up a jsfiddle).

**Comment by gonzalle** [comment]
Thx for your answer... My purpose is to have two transform animations on the same object... but with different durations and easings. For instance a translate that last 2 secs and, in the same time, a scaling that last 4 seconds. The only way around with Snap is creating a group... 
See this [codepen](http://codepen.io/viotech/pen/WpMrXg) where I compare solutions with an other library like anime.js (that I don't want to use)...

**Comment by ibrierley** [comment]
What other problems do adding an extra group add ? Just wondering as any other solution may be a bit hacky and introduce the same problem.

**Comment by clementgonzalez** [comment]
I just needed to keep my SVG DOM untouched...

**Comment by ibrierley** [comment]
Well you could try something like http://codepen.io/anon/pen/RpQoRJ

That runs 2 separate animate functions, but takes 2 separate figures and combines them.

**Comment by clementgonzalez** [comment]
great solution for me ! It's indeed a little complex but it works. This option might be included into the library !  Thanks a lot...
